### PR TITLE
fix ui::WebView callback operate OpenGL issue.

### PR DESCRIPTION
**Pull request by zilongshanren**
All the OpenGL operations should be in OpenGL thread.
This PR wraps the callbacks in OpenGL thread.

### CODE REVIEW

**Comment by zilongshanren** [comment]
@WenhaiLin  @pandamicro  @dumganhar  please review.

File: cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java [hunk 1]
```diff
@@ -58,29 +58,51 @@ public void setScalesPageToFit(boolean scalesPageToFit) {
     class Cocos2dxWebViewClient extends WebViewClient {
         @Override
         public boolean shouldOverrideUrlLoading(WebView view, String urlString) {
+            Cocos2dxActivity activity = (Cocos2dxActivity)getContext();
+            final String finalizedUrlString = urlString;
```
Comment (dumganhar) [inline cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java]: Why not to make url argument as 'final' string？

Comment (zilongshanren) [inline cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java]: It‘s an overrided method, is it Ok to change the arguments?

**Comment by pandamicro** [comment]
[ci rebuild]

**Comment by pandamicro** [comment]
I think it's ok, @dumganhar what do you think

File: cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java (hunk 1 above)
Comment (dumganhar) [inline cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java]: Sure，`final` keyword will not affect the `override` status.

Comment (zilongshanren) [inline cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java]: @dumganhar Done, please check.
Thanks.

**Comment by pandamicro** [comment]
[ci rebuild]

**Comment by dumganhar** [comment]
@pandamicro , It's ok for merge.

**Pull request merged**